exchange-contract/docs/notebooks/index.ipynb

```diff
@@ -0,0 +1,122 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "67bb03c9-aa81-4491-bf6f-ce9a51d1af7d",
+   "metadata": {},
+   "source": [
+    "# Exchange Contract Family\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d656d82-daec-42fa-8d89-67c45f09f9e1",
+   "metadata": {},
+   "source": [
+    "## Getting Started\n",
+    "\n",
+    "Assumptions: \n",
+    "* PDO client virtual environment has been installed in `${PDO_INSTALL_ROOT}`\n",
+    "* PDO Contracts packages have been built and installed in the client virtual environment\n",
+    "* PDO services are running \n",
+    "\n",
+    "1. Create a personal configuration file\n",
+    "2. Create a services database\n",
+    "3. Create a service group configuration"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ee5e4693-c43d-4a6a-bb95-583f75643780",
+   "metadata": {},
+   "source": [
+    "### Simplified configuration\n",
+    "\n",
+    "If you are running the PDO services in the common test configuration (e.g. with 5 provisioning services, 5 enclave service, and 5 storage services all running on the same host, you can create all of the necessary configuration files with the command:\n",
+    "    \n",
+    "```\n",
+    "    $ export service_host=<service hostname>\n",
+    "    $ ${PDO_INSTALL_ROOT}/opt/pdo/bin/pdo-create-service-groups.psh --service_host ${service_host}\n",
+    "```\n",
+    "\n",
+    "Use `service_host` in the notebooks that you create.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "bc2fe555-a933-42e4-ad25-421e56baaac5",
+   "metadata": {},
+   "source": [
+    "### Create a personal configuration file\n",
+    "\n",
+    "The default PDO client configuration file is in `${PDO_INSTALL_ROOT}/opt/pdo/etc/pcontract.toml`. This file contains a basic configuration that should suffice for testing on a single host. \n",
+    "\n",
```

Comment:
not sure i understand this. you already above gave the command to create the groups file based on default configuration. are you saying that "a future notebook to override the default config, and create the groups file based on custom config" ?